shrs/src/shell.rs

```diff
@@ -132,6 +134,7 @@ impl ShellConfig {
             args: vec![],
             exit_status: 0,
             functions: self.functions,
+            timer: ProcessTimer::new(),
```

Comment:
remove process timer from `Runtime` and replace with just init time (when you want to get time since start, just call elapsed)